src/main/java/monitor/MonitoringRunner.java

```diff
@@ -29,6 +43,11 @@ public static void startServer() {
         }
     }
 
+    /**
+     * Method that looks up a list of JobRecord objects in the DB
+     * for a specific date and formats an appropriate response in
```

Comment:
It returns all record starting from specific date (10 days prior to request time)